question: Cannot find module '@apitable/core' or its corresponding type declarations.

make run
select 2 to start room-server
Cannot find module '@apitable/core' or its corresponding type declarations.
help~thanks

![errorimg](https://user-images.githubusercontent.com/54765789/217420601-517ed05f-9381-4ede-b16b-5d7729d4e843.png)

--- a/Makefile
+++ b/Makefile
@@ -309,7 +309,6 @@ _run-local-web-server:
 	source scripts/export-env.sh $$ENV_FILE;\
 	source scripts/export-env.sh $$DEVENV_FILE;\
 	rm -rf packages/datasheet/web_build;\
-	yarn build:dst:pre;\
 	yarn sd
 
 _run-local-socket-server:

